Common/src/gRPC/Services/Agent.cs

```diff
@@ -644,6 +642,152 @@ await completionTask.WaitAsync(cancellationToken)
            };
   }
 
+  /// <inheritdoc />
+  public async Task<SubmitTasksResponse> SubmitTasks(SubmitTasksRequest request,
```

Comment:
I am not sure why do you need to duplicate those methods from submitter.

Reply:
because this one does not have to manage payload data